src/Files.App/Views/Shells/BaseShellPage.cs

```diff
@@ -137,9 +137,6 @@ public bool IsCurrentInstance
 				{
 					_IsCurrentInstance = value;
 
-					if (!value && SlimContentPage is not ColumnsLayoutPage)
-						ToolbarViewModel.IsEditModeEnabled = false;
-
```

Comment:
Is there a way we can force close the Omnibar? This would allow us to solve the issue where its sometimes open after switching tabs.

Reply:
@0x5bfa